bulletin/index.html

```diff
@@ -203,6 +203,53 @@ <h2 class="featured-article__title"> Engineering Robust Learning Environments Th
 	  </div>
 	</section>
 
+	<section class="mention">
+		<div class="mention-container mention-container__wrapper">
+			<div class="mention-heading">
```

Comment:
`mention-heading` could be a `header` instead of a `div`. It would be the most semantic element given the context